### ISSUE TITLE
Replacing the existing code with React.js with UI enhancements

### ISSUE THREAD

**Issue by debianbaker** [rajatuiwebdev/badge-website#1]
We can switch to React components with Tailwind structure rather than Html and Css which is according to the current standards.

**Comment by debianbaker** [comment]
Waiting for review :) @rajatuiwebdev

**Comment by rajatuiwebdev** [comment]
Assigned. Good luck.

**Comment by Abhineetsahay** [comment]
I added ReactJs and tailwind can give a look to it.

**Comment by rajatuiwebdev** [comment]
Yes, @Abhineetsahay I saw your submission but it does not overwrite the current code and hence I can not merge it. I've opened other issues, see if you can contribute there.

**Comment by Abhineetsahay** [comment]
ok @rajatuiwebdev

**Comment by Kedarnath-Rothe** [comment]
Please assign this issue to me i can change This from HTML to React... as i have  completed various projects on MERN.....

**Comment by rajatuiwebdev** [comment]
Hello @debianbaker once I merge all the PRs, feel free to convert the latest branch to React and then I'll merge it.
